src/parse/asp/interpreter.go

```diff
@@ -38,15 +39,19 @@ func newInterpreter(state *core.BuildState, p *Parser) *interpreter {
 	}
 	// If we're creating an interpreter for a subrepo, we should share the subinclude cache.
```

Comment:
This isn't actually useful any more, although I haven't experimented with whether it's better or worse if we share / don't share them.
(It obviously is very useful for the asts though)